src/components/AssetMake/AddAssetMakeComponent.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form } from 'semantic-ui-react';
+import ArtButton from '../common/ButtonComponent';
+import InputFluid from '../common/TextInputComponent';
+import DropdownComponent from '../common/DropdownComponent';
+import '../../_css/AddAssetComponent.css';
+
+const assetTypeOptions = assetTypes =>
+  assetTypes && assetTypes
```

Comment:
This check will always be true as an empty array is a truthy value. Just use `assetTypes.map()` and ensure that the default value of `assetTypes` will always be an empty array, not undefined

Reply:
I had used that because at first the assetTypes is undefined. I will add its default value.